### PR TITLE
Add feature View command

### PR DESCRIPTION
**Pull request by nigelteosw**

### CODE REVIEW

**Comment by nigelteosw** [comment]
todo: Add testcase for when food does not exist in list

File: src/main/java/seedu/wife/logic/commands/foodcommands/ViewCommand.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            + ": Views the food identified by the index number used in the displayed food list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_VIEW_FOOD_SUCCESS = "Viewed Food: %1$s";
+
+    private final Index targetIndex;
+
+    public ViewCommand(Index targetIndex) {
+        this.targetIndex = targetIndex;
```
Comment (Jnjy) [inline src/main/java/seedu/wife/logic/commands/foodcommands/ViewCommand.java:32]: Should you have a `requireNonNull` here to make sure the inserted `targetIndex` is not a null value?

File: src/main/java/seedu/wife/logic/commands/foodcommands/ViewCommand.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Food> lastShownList = model.getFilteredFoodList();
+
+        if (targetIndex.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_FOOD_DISPLAYED_INDEX);
```
Comment (Jnjy) [inline src/main/java/seedu/wife/logic/commands/foodcommands/ViewCommand.java:41]: Do you think it's better to have `MESSAGE_INVALID_FOOD_DISPLAYED_INDEX` imported with static import so it is consistent with line 46? 

Same issue exists for `ViewCommandTest`

File: src/main/java/seedu/wife/logic/commands/foodcommands/ViewCommand.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        }
+
+        Food foodToView = lastShownList.get(targetIndex.getZeroBased());
+        model.viewFood(foodToView);
+        return new CommandResult(String.format(MESSAGE_VIEW_FOOD_SUCCESS, foodToView));
+    }
+
```
Comment (Jnjy) [inline src/main/java/seedu/wife/logic/commands/foodcommands/ViewCommand.java:48]: Perhaps you can remove the extra new line here

**Review by Jnjy — CHANGES_REQUESTED** [review]
Overall the logic is good, just some minor changes required for code quality.

File: src/main/java/seedu/wife/logic/commands/foodcommands/ViewCommand.java (hunk 2 above)
Comment (nigelteosw) [reply src/main/java/seedu/wife/logic/commands/foodcommands/ViewCommand.java:42]: wdym by static import for that? it is the same functionality as delete though

Comment (Jnjy) [reply src/main/java/seedu/wife/logic/commands/foodcommands/ViewCommand.java:45]: I meant like `import static seedu.wife.commons.core.Messages.MESSAGE_INVALID_FOOD_DISPLAYED_INDEX`, its more of keeping the code standardized but it is okay for now

**Review by Jnjy — APPROVED** [review]
LGTM

File: src/main/java/seedu/wife/logic/commands/foodcommands/ViewCommand.java (hunk 2 above)
Comment (nigelteosw) [reply src/main/java/seedu/wife/logic/commands/foodcommands/ViewCommand.java:45]: okay will keep that in mind next time

**Pull request merged**